### ISSUE TITLE
[ Proposal ] Freehand Mode

### ISSUE THREAD

**Issue by Soare-Robert-Daniel** [Codeinwp/logo-maker#36]
## Introduction

The current layout settings on Logo Maker are based on predefined templates: TOP, LEFT RIGHT. I suggest adding one more option: Freehand - in which the user can drag the SVG to the desired positions.

Preview: 
![image](https://user-images.githubusercontent.com/17597852/113406227-cba5c580-93b3-11eb-83dd-8a3143a2b4a2.png)

## Implementation

For dragging the SVGs, we can use native functions based on mouse events - you can have a peak in this [tutorial](https://www.petercollingridge.co.uk/tutorials/svg/interactive/dragging/).

Besides the drag option, it will be nice to have a way to scale ( only the logo, slogan & text have this option via `font-size` in Typography ) or rotate. Scaling and rotating the SVG might some problems for the user with small screens. Unlike dragging which occurs on the mouse, these options will available via floating element - they will appear when you select an element.

Preview:
![image](https://user-images.githubusercontent.com/17597852/113407746-9b135b00-93b6-11eb-970f-cb29efb6c4dc.png)

![image](https://user-images.githubusercontent.com/17597852/113408517-2a6d3e00-93b8-11eb-844e-372043953f14.png)

@ineagu, @HardeepAsrani, @selul, @mghenciu, @mileamadalin  what do you think?

**Comment by mghenciu** [comment]
@Soare-Robert-Daniel 
Having the option to move the Logo sounds really good, especially for the cases where you need to [optically align elements](https://ibb.co/mFsWtCV). Same thing for the rotation, it's a good idea, maybe for now it's worth adding it only for the logo (rotating the text I don't think has a lot of popularity).

Something to keep in mind about the moving option, is that usually you'll want to make small changes (in terms of moving distance), and make the moves as precise/frictionless as possible, that's why most of the time I use snapping options or I move the elements with arrow keys, knowing that each key tap has 8px. 
Long story short, make sure that items move slowly (than usual) when you drag them.
